examples/tutorials/sklean_models.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<p>Make the function more universal and put it into <code>batchflow.utils</code></p><p> </p><p>It could also be a good idea to compare log reg and SVM in terms of various metrics and particular predictions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>